postgres-appliance/Dockerfile

```diff
@@ -382,8 +382,9 @@ RUN export DEBIAN_FRONTEND=noninteractive \
 \
     && if [ "$DEMO" != "true" ]; then \
         EXTRAS=",etcd,consul,zookeeper,aws" \
-        && curl -sL https://github.com/wal-g/wal-g/releases/download/$WALG_VERSION/wal-g.linux-amd64.tar.gz \
+        && curl -sL https://github.com/wal-g/wal-g/releases/download/$WALG_VERSION/wal-g-pg-ubuntu-18.04-amd64.tar.gz \
                 | tar -C /usr/local/bin -xz \
+        && mv /usr/local/bin/wal-g-pg-ubuntu-18.04-amd64 /usr/local/bin/wal-g \
```

Comment:
```suggestion
        && DISTRIB_RELEASE=$(sed -n 's/DISTRIB_RELEASE=//p' /etc/lsb-release) \
        && curl -sL https://github.com/wal-g/wal-g/releases/download/$WALG_VERSION/wal-g-pg-$DISTRIB_RELEASE-amd64.tar.gz \
                | tar -C /usr/local/bin -xz \
        && mv /usr/local/bin/wal-g-pg-$DISTRIB_RELEASE-amd64 /usr/local/bin/wal-g \
```

Reply:
Using /etc/lsb-release seems like it could be fragile -- are you sure you want to make this change?